python/sparkdl/graph/input.py

```diff
@@ -0,0 +1,262 @@
+# Copyright 2017 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from __future__ import absolute_import, division, print_function
+
+import tensorflow as tf
+from tensorflow.core.protobuf import meta_graph_pb2  # pylint: disable=no-name-in-module
+
+import sparkdl.graph.utils as tfx
+
+__all__ = ["TFInputGraph"]
+
+# pylint: disable=invalid-name,wrong-spelling-in-docstring,wrong-spelling-in-comment
+
+
+class TFInputGraph(object):
+    """
+    An opaque object containing TensorFlow graph.
+    This object can be serialized.
+
+    .. warning: This class should not be called by any user code.
```

Comment:
document fields here (mentioning they are implementation details that should be not be relied on by users).

Reply:
Expanded documentation.